src/processors/default/processPlatformTracks/processPlatformTracks.ts

```diff
@@ -62,11 +63,12 @@ const processorFunction = (platform: MusicPlatform) => async (nfts: NFT[], clien
     let nftFactoryType: MusicPlatformTypeConfig;
     try {
       nftFactoryType = getNFTFactoryType(nftFactory, platformType);
+      const extractor = idExtractor(nftFactory);
```

Comment:
Chose to always invoke and return undefined when not required so it is backward compatible for any platform